File: C++/Medium/WrongAnswer/clumsy-factorial_6.cpp
class Solution {
public:
    int clumsy(int n) {
        
        int result = 0;
        int temp = 0;
        int i = 0;
        while (n > 0) {
            if (i == 0) {
                temp = n;
            } else if (i == 1) {
                temp *= n;
            } else if (i == 2) {
                temp /= n;
            } else if (i == 3) {
                result += temp;
                temp = n;
            } else {
                result -= temp;
                temp = n;
            }
            n--;
            i++;
            if (i == 4) {
                i = 0;
            }
        }
        if (i == 0) {
            result += temp;
        } else if (i == 1) {
            result += temp;
        } else if (i == 2) {
            result += temp;
        } else if (i == 3) {
            result += temp;
        }
        return result;
    }
};

File: Java/Medium/WrongAnswer/clumsy-factorial_6.java
class Solution {
    public int clumsy(int n) {
        
        int[] arr = new int[n];
        for(int i = 0; i < n; i++){
            arr[i] = n - i;
        }
        int result = arr[0];
        int i = 1;
        while(i < n){
            result = result * arr[i];
            i++;
            if(i < n){
                result = result / arr[i];
                i++;
            }
            if(i < n){
                result = result + arr[i];
                i++;
            }
            if(i < n){
                result = result - arr[i];
                i++;
            }
        }
        return result;
    }
}